xds/third_party/envoy/import.sh

```diff
@@ -17,7 +17,7 @@
 
 set -e
 # import VERSION from the google internal copybara_version.txt for Envoy
-VERSION=ab911ac2ff971f805ec822ad4d4ff6b42a61cc7c
+VERSION=fcdc9d6d5a9da98fe30ad749201034770ff6b30e
```

Comment:
This version changed and no proto (other than maybe gcp_authn.proto) changed? That is very unlikely. And it seems you've updated four months of changes. Yeah, there's no way no other proto changed. You need to re-run the import.sh script.

And this version is too new. See the comment just above the line. It should be no newer than 742a3b02e3b2a9dfb877a7e378607c6ed0c2aa53 .

What change are you updating this version for? I'll note it is often useful to upgrade envoy protos as a separate PR as it is easy to causes a bunch of noise and easy to cause conflicts. Doing it in a separate PR means you can merge the change quickly, and thus reduce how many other changes could cause conflicts.

Reply:
Oh, I see. I took the latest commit available. Need to remove all this from this PR and create a new one, taking latest from copybara import and not github.

Reply:
> Why this is dependent on copybara import?

It doesn't have to, but it makes our life much easier. If we depend on something that hasn't been imported yet, then we need to push for the import and our own imports are blocked. So it is much better to get internal upgraded before we depend on it in gRPC.

This sort of thing applies to any dependency, but most dependencies we aren't depending on new functionality. But let's say we needed an upgrade to the auth library to use a new class, then it would be appropriate to upgrade that internally before we depend on the new code here on github.

The deps that are the highest risk for issue are envoy protos, Netty, and netty-tcnative.